### PR TITLE
docs(esl-traversing-query): improve documentation for esl-traversing-query

### PR DESCRIPTION
**Pull request by HenadzV**
<body>
add note recarding scope, missing examples

Closes: #1864

### CODE REVIEW

**Review by ala-n — APPROVED** [review]

File: src/modules/esl-traversing-query/README.md [hunk 1]
```diff
@@ -6,17 +6,18 @@ Authors: *Alexey Stsefanovich (ala'n)*.
...
  - plain CSS selectors
- - relative selectors  
+ - relative selectors
 NOTE: Selectors that don't start from a plain selector will use passed base Element as a root
  - `::next` and `::prev` sibling pseudo-selectors
  - `::parent`, `::closest` and `::child` pseudo-selectors
  - `::find` pseudo-selector
  - `::first`, `::last` and `:nth(#)` limitation pseudo-selectors
  - `::visible` visible element pseudo-selector
+ - `::not`, `::filter` filtration pseudo-selectors
```
Comment (abarmina) [inline src/modules/esl-traversing-query/README.md:20]: as a suggestion - do we need to specify here which ones we just apply and which ones we pass "arguments" to

File: src/modules/esl-traversing-query/README.md [hunk 2]
```diff
@@ -51,3 +58,7 @@ Params:
 - `query` - query string
 - `base` - describes a base for relative query.
 - `scope` - defines a scope to find global query (`document` by default)
+
+---
+
+With the ESL Traversing Query, you can harness the power of extended selector queries to perform sophisticated DOM traversals and element selections. Whether you're targeting specific elements, navigating the DOM tree, or filtering results, this utility simplifies complex tasks and enhances your ability to manipulate the webpage's structure and content.
```
Comment (abarmina) [inline src/modules/esl-traversing-query/README.md:64]: can we simplify this word a little? but maybe I just don't know it :smiley:
```suggestion
With the ESL Traversing Query, you can harness the power of extended selector queries to perform complicated DOM traversals and element selections. Whether you're targeting specific elements, navigating the DOM tree, or filtering results, this utility simplifies complex tasks and enhances your ability to manipulate the webpage's structure and content.
```

**Review by abarmina — APPROVED** [review]

File: src/modules/esl-traversing-query/README.md [hunk 3]
```diff
@@ -31,9 +32,15 @@ NOTE: Selectors that don't start from a plain selector will use passed base Elem
 - `::closest(#id .class [attr])` - find the closest ancestor including base element that matches passed selector
 - `::child(#id .class [attr])` - find direct child element(s) that match passed selector
 - `::find(#id .class [attr])` - find child element(s) that match passed selector
+- `::find(buttons, a)::not([hidden])` - find all buttons and anchors that are not have hidden attribute
```
Comment (NastaLeo) [inline src/modules/esl-traversing-query/README.md:35]:
```suggestion
- `::find(buttons, a)::not([hidden])` - find all buttons and anchors that don't have hidden attribute
```

**Review by NastaLeo — APPROVED** [review]

File: src/modules/esl-traversing-query/README.md (hunk 1 above)
Comment (HenadzV) [reply src/modules/esl-traversing-query/README.md:20]: I don't think so, from examples, it's quite obvious. We can discuss it  on the Monday call

**Pull request merged**